src/bpmncwpverify/builder/bpmn_builder.py

```diff
@@ -20,7 +19,7 @@ def build(self) -> Result[Bpmn, Error]:
             validate_bpmn(self._bpmn)
             return Success(self._bpmn)
         except Exception as e:
-            return Failure(ExceptionError(str(e)))
+            return Failure(str(e))
```

Comment:
Why is this just a string? It should be a specific error type that sets appropriate fields that are then turned to a string in our `get_error_message` function.